### ISSUE TITLE
[Dividers] components are read out loud by screen readers

### ISSUE THREAD

**Issue by theetrain** [telus/tds-core#689]
## Description

Divider components, including hairline, wave, and dimple, all lack the aria-hidden attribute. When focusing on page content, the dividers should be skipped when using Screen Reader gestures.

As a result, the voice over gesture feature on MacOS reads these components out loud when they should be skipped instead.

## Reproduction Steps

1. Activate macOS screen reader commander (command + F5, then hold control + option and use two fingers to make a clockwise gesture on the touch pad)
2. Navigate through a page that has a divider by using a single finger swipe to the left or right
3. Observe the divider is read out loud when hovered upon

Note: to disable commander, use control + option and then gesture counter-clockwise
## Reproduction Percentage (if applicable)

* 10/10 times

## Meta

* TDS version: All versions of the divider components
* Willing to develop solution: Yes
* Has workaround: Yes, you can explicitly add the `aria-hidden` attribute to any divider, but it would be preferred to have this enabled by default
* High impact: No

## Screenshots

![image](https://user-images.githubusercontent.com/12798751/42830070-98ab1d36-89b8-11e8-843b-9f142c098e6c.png)

**Comment by theetrain** [comment]
Screen readers have the ability to navigate around `<hr>` elements. Having our dividers always be hidden may not be desirable. Let's consider the following:

- Add an optional `a11yHide` prop that sets `aria-hidden="true"` when used
- Consider always hiding the hairline divider when used in the ExpandCollapse component
- See if there are ways to label the dividers so that screen readers can identify sections

Thanks @ah-arch 👋

**Comment by jraff** [comment]
@owestinTelus Could you follow up with a standard for accessible dividers?

**Comment by owestinTelus** [comment]
Issue will be reviewed later this year.
I have created a JIRA ticket in the accessibility backlog.
https://telusdigital.atlassian.net/browse/A11Y-100

This ticket can be closed.

**Comment by theetrain** [comment]
Closing for now until this topic resurfaces.
